lib/cli/lib/.babelrc

```diff
@@ -1,9 +0,0 @@
-{
-  "presets": ["env", "stage-0", "react"],
-  "plugins": [
-    ["transform-runtime", {
-      "polyfill": false,
-      "regenerator": true
-    }]
-  ]
-}
```

Comment:
I believe this file was added by mistake: https://github.com/storybooks/storybook/pull/911/files#diff-c2e6a73df4341abfc1cce840a8008e9e